test/unit/sandbox-libraries/pm.test.js

```diff
@@ -19,6 +19,14 @@ describe('sandbox library - pm api', function () {
                 value: 2.5,
                 type: 'number'
             }],
+            collectionVariables: [{
+                key: 'var1',
+                value: 'collection-var1',
+                type: 'string'
+            }, {
+                key: 'var3',
+                value: 'collection-var3'
```

Comment:
Any specific reason for skipping the `type` here?

Reply:
yup, I threw in some of these randomly to make sure legacy variables work